src/typings/index.d.ts

```diff
@@ -0,0 +1,35 @@
+type PokemonCardObject = {
```

Comment:
```suggestion
type PokemonCard = {
```

No need to get too in the weeds with it